views/routes/routes.js

```diff
@@ -16,7 +16,7 @@ export const routes = {
   charts: `${basePath}/charts`,
   reports: `${basePath}/reports`,
   admin: `${basePath}/admin`,
-  login: `${basePath}/login`,
+  login: `/login`,
```

Comment:
why are we removing the basePath here but nowhere else?

Reply:
Those changes belong to this pr. https://github.com/AMP-SCZ/dpdash/pull/319

Reply:
I'm not sure I follow. So you're saying this diff shouldn't be in this PR and is going to be removed?

Reply:
ok let's move this diff over to #319 then so this PR is isolated to the account expires bug.

Reply:
This goes together, after logout we reroute to login

Reply:
right, but that path already exists. Are you saying the path needs to change in order for this to work? I think the base path is empty anyway so this doesn't break/fix the path.

Reply:
Yes I feel like this will break in a deployed instance, because as the application is being deployed the basePath file is being written on the fly and I don't know what those values could be. To avoid the risk of it not working it is in our best interest to remove it.  https://github.com/AMP-SCZ/dpdash/blob/bf46996b5d49c8e60584c290779f1752a942061c/singularity/configure.py#L416-L424